ktor-http/ktor-http-cio/js/src/io/ktor/http/cio/websocket/FrameJs.kt

```diff
@@ -14,19 +14,29 @@ import io.ktor.utils.io.core.*
  * @property data - a frame content or fragment content
  * @property disposableHandle could be invoked when the frame is processed
  */
-public actual sealed class Frame private actual constructor(
+public actual sealed class Frame actual constructor(
     public actual val fin: Boolean,
     public actual val frameType: FrameType,
     public actual val data: ByteArray,
-    public actual val disposableHandle: DisposableHandle
+    public actual val disposableHandle: DisposableHandle,
+    public actual open val rsv1: Boolean,
+    public actual open val rsv2: Boolean,
+    public actual open val rsv3: Boolean
```

Comment:
Shouldn't we add init block checking rsv flags conflict? It shouldn't be that slow in spite of that it's done for every frame

Reply:
Discussed